exist-core/pom.xml

```diff
@@ -589,6 +589,16 @@
             <artifactId>assertj-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Do we really need to add another Mocking framework? Is it not possible to use EasyMock here (there is already code dependent on that)

Reply:
Any way to mock non interfaces with EasyMock?

Reply:
Yes. It used to be an extension, but I think it is just built-in in modern versions of EasyMock.

Reply:
One such existing example of this - https://github.com/eXist-db/exist/blob/eXist-6.2.0/exist-core/src/test/java/org/exist/storage/journal/JournalTest.java#L282